### PR TITLE
add support to grpc 1.1.2

### PR DESCRIPTION
**Pull request by overstart**

### CODE REVIEW

**Comment by fanminshi** [comment]
etcd still uses grpc-go 1.04,https://github.com/coreos/etcd/blob/master/glide.yaml#L100. I don't want to upgrade to a grpc version that's higher than that of etcd yet.

**Comment by xiang90** [comment]
@fanminshi I am not sure how java grpc version their client. Can you check if java 1.1 is suggested to work with grpc server 1.1 only, not 1.0?

**Comment by fanminshi** [comment]
I didn't find anything from google search. So I created a issue on grpc-go, https://github.com/grpc/grpc-java/issues/2844

**Comment by fanminshi** [comment]
@xiang90 grpc people says that 

"The versioning of gRPC is called semantic versioning. All major versions, i.e. the leading number, that are the same will work together."

Also the latest is grpc-java 1.2.0, https://github.com/grpc/grpc-java/releases/tag/v1.2.0 but uses protobuf 3.2.0. We might want to upgrade to the latest.

**Comment by xiang90** [comment]
sgtm.

File: pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
         <protobuf-maven-plugin.version>0.5.0</protobuf-maven-plugin.version>
 
         <!-- dependencies -->
-        <grpc.version>1.0.0</grpc.version>
+        <grpc.version>1.1.2</grpc.version>
```
Comment (fanminshi) [inline pom.xml]: Could you use 1.2.0 for `grpc.version`and 3.2 for `protobuf.version`?

**Comment by fanminshi** [comment]
can you also change the git commit message to
`pom: add support to grpc 1.2.0 and protobuf 3.2.0`

**Comment by fanminshi** [comment]
close via #77

**Pull request closed without merging**